code/__DEFINES/tgs.dm

```diff
@@ -10,6 +10,10 @@
 //Comment this out once you've filled in the below
 #error TGS API unconfigured
 
+//Uncomment this if you wish to allow the game to interact with TGS 3
+//This will raise the minimum required security level of your game to TGS_SECURITY_TRUSTED due to it utilizing call()()
+//#define TGS_V3_API

```

Comment:
This is here but it isn't used anywhere in the rest of the diff at all. Did you mirror the full change or just this file?